ironfish/src/syncer.ts

```diff
@@ -484,6 +484,11 @@ export class Syncer {
 
     const { added, block } = await this.addBlock(peer, newBlock)
 
+    peer.knownBlockHashes.set(block.header.hash, true)
+    for (const knownPeer of peer.knownPeers.values()) {
+      knownPeer.knownBlockHashes.set(block.header.hash, true)
```

Comment:
Should we add this to known peers as well? Especially for blocks it seems like if some of these peers are still syncing it's possible they haven't see this block hash yet. I was not planning on doing this for transactions